crates/router/src/connector/worldline/transformers.rs

```diff
@@ -174,6 +174,7 @@ pub struct PaymentsRequest {
     pub payment_data: WorldlinePaymentMethod,
     pub order: Order,
     pub shipping: Option<Shipping>,
+    pub reference: String,
```

Comment:
Hello @Kota-Karthik ,

In reference to issue #2351, we need to adjust the reference field mapping based on the API documentation of the connector. The merchant reference ID should be structured as follows: `order` -> `references` -> `merchantReference` for the request . For the response, `connector_response_reference_id` should be mapped to  `paymentOutput` -> `references` -> `merchantReference`. Kindly make the required modifications based on their documentation. Additionally, there were some unnecessary files pushed in your recent commits; could you please remove them?

You can find the reference for Worldline API documentation [here](https://apireference.connect.worldline-solutions.com/s2sapi/v1/en_US/java/payments/create.html?paymentPlatform=ALL).

Thank you.